sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -66,7 +66,15 @@ FAKE_PROD_KEYS = ["fake_prod_key_0"]
 
 FAKE_ALL_KEYS = FAKE_TEST_KEYS + FAKE_DEV_KEYS + FAKE_PROD_KEYS
 
-# list of all acceptable keys in each LC state
+# Specify the ROM key slot
+# FIXME this might not be the best place/way to have this information
```

Comment:
I have updated the PR to do that (partially). Since there is no notion of keyset/key domain yet, there is still some annoying coupling between `rules/opentitan.bzl` that holds the dictionary of key names, and `rom_keys.bzl` that holds some generic names like `test_0` but that need to be "adjusted" given the key set name (`fake`) to give the final key name (`fake_test_0`). This is why there is `keymap_for_keyset` function at the moment.
Solving this would probably be best done as part of another PR, e.g. see #17339

Reply:
I have used the key slot documented in #17474, keep in sync.